modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/workflow-builder/WorkflowStep.es.js

```diff
@@ -20,14 +20,25 @@ import React, {useState} from 'react';
 import ButtonInfo from '../../../../components/button-info/ButtonInfo.es';
 
 const Arrow = ({addStep, selected}) => {
+	const isIE = navigator.userAgent.indexOf('Trident/7.0;') != -1;
```

Comment:
Hey @naraandrade,

https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-jquery-web/src/main/resources/META-INF/resources/jquery/form.js#L1561 is that enough?

Reply:
or https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-jquery-web/src/main/resources/META-INF/resources/jquery/form.js#L1273

Reply:
@inacionery I will check, thank you :)